src/lib.rs

```diff
@@ -50,6 +50,50 @@ pub const TLS_SERVER_ROOTS: &[TrustAnchor] = &[
     name_constraints: None
   },
 
+  /*
```

Comment:
Is there a reason for not cherry-picking my commit here? Feels like that might help establishing that the same changes were made in both branches?

Reply:
No particular reason. I'll adjust :+1:

Reply:
Adjusted to use a cherry-pick. I had to manually fix the usage of pki-types + webpki API changes, so it's not a 1:1 with the commit from your `main` but there's no avoiding that.